packages/ui/src/Help.tsx

```diff
@@ -14,19 +14,27 @@ export const helpTooltipId = (inputId: string): string => `${inputId}-help`
 export interface HelpProps {
   parentId: string
   helperText: string | ReactElement
-  placement?: string
+  placement?: TooltipProps['placement']
   className?: string
   padding?: 'default' | 'none'
 }
 
-export const Help: FC<HelpProps> = ({ helperText, placement = 'top', parentId, className, padding = 'default' }) => (
-  <Tooltip placement={placement} overlay={helperText} id={helpTooltipId(parentId)}>
-    <div
-      className={cn(styles.container, className, {
-        [styles.padding]: padding === 'default',
-      })}
-    >
-      <Icon ariaLabel="Help" color={styleConsts.colorPrimary} icon={HelpCircle} className={styles.icon} size="small" />
-    </div>
-  </Tooltip>
-)
+export const Help: FC<HelpProps> = ({ helperText, placement = 'top', parentId, className, padding = 'default' }) => {
+  const tooltipId = helpTooltipId(parentId)
+
+  return (
+    <Tooltip placement={placement} content={helperText} id={tooltipId}>
+      {(ref) => (
+        <div
+          ref={ref}
+          className={cn(styles.container, className, {
+            [styles.padding]: padding === 'default',
+          })}
+          aria-describedby={tooltipId}
```

Comment:
nit: Should we move it to the `Icon`? I feel like we are describing the icon actually